README.md

```diff
@@ -160,10 +162,16 @@ mvn test
 - Ask us on [stackask.com](https://stackask.com)
 - Email us at [Dzung](mailto:itprono3@gmail.com)
 
-# Help us by making a donation.
+# Support Us: Make a Meaningful Donation.
```

Comment:
Seems like we're not being case consistent. E.g.,
- Latest version (lower case 'v')
- Tutorial: (redundant ':')

Could you revise all other headings in this MR as well? Thanks

Reply:
Fix all headings to be case consistent